cmd/federation-controller/main.go

```diff
@@ -24,17 +24,31 @@ import (
 	"syscall"
 	"time"
 
-	v1 "k8s.io/client-go/listers/core/v1"
-
 	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 	routev1client "github.com/openshift/client-go/route/clientset/versioned"
 	istiokube "istio.io/istio/pkg/kube"
 	istiolog "istio.io/istio/pkg/log"
 	"istio.io/istio/pkg/slices"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/informers"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	v1 "k8s.io/client-go/listers/core/v1"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	// +kubebuilder:scaffold:imports
```

Comment:
As long as we do not intended to use scaffolding by calling

```
kubebuilder create api|webhook
```

having any of `// +kubebuilder:scaffold:*` markers is just a noise in the code.

Reply:
This is how I created the controller, so I wasn't sure if we should remove it or not...?